### PR TITLE
Paths

### PR DESCRIPTION
**Pull request by ethn**
Some cleanup of path and config handling.  Needs comments!

### CODE REVIEW

File: card/db/migrate_core_cards/20141230204340_uri_codename.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # -*- encoding : utf-8 -*-
 
-class UriCodename < Card::Migration
```
Comment (ethn) [inline card/db/migrate_core_cards/20141230204340_uri_codename.rb]: doh, I'd merged in Philipp's migration changes.

File: card/lib/cardio.rb [hunk 2]
```diff
@@ -11,55 +11,71 @@ module Cardio
...
+      add_gem_path "db/seeds",            :with => "db/seeds.rb"
+
+      add_gem_path 'config/initializers', :glob => '**/*.rb'
     end
 
+#    def run_initializers
+#      paths['config/initializers'].existent.sort.each do |initializer|
+#        load(initializer)
+#      end
+#    end
```
Comment (ethn) [inline card/lib/cardio.rb]: these get run by wagn now.  need to resolve this for rails decks.

**Comment by ethn** [comment]
arg. the assets file name changes (moved from wagn to decko-rails) obscured the key changes here.  I may need to redo this.

**Comment by ethn** [comment]
the most important file is this: https://github.com/ethn/wagn/blob/paths/wagn/lib/wagn/application.rb

You can tell at a glance that it's pretty different.

**Comment by GerryG** [comment]
I tested the migrations again and they work without any help like before.  (on this branch, not yet on 1_15)

**Comment by ethn** [comment]
very cool.  thanks for doing that!

**Pull request merged**